lib/rules/prefer-node-protocol.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @author Yusuke Iinuma
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const isBuiltinModule = require("is-builtin-module")
+const getConfiguredNodeVersion = require("../util/get-configured-node-version")
+const getSemverRange = require("../util/get-semver-range")
+const visitImport = require("../util/visit-import")
+const visitRequire = require("../util/visit-require")
+const mergeVisitorsInPlace = require("../util/merge-visitors-in-place")
+
+const messageId = "preferNodeProtocol"
+
+module.exports = {
+    meta: {
+        docs: {
+            description:
+                "enforce using the `node:` protocol when importing Node.js builtin modules.",
+            recommended: true,
```

Comment:
the `recommended` config changes would be a breaking change.
```suggestion
            recommended: false,
```

Reply:
Thank you for your review 🙏 
I fixed it.